src/configuration/app/multi-app.md

```diff
@@ -31,231 +24,141 @@ angular/
-        index.php
-app2/
-    .platform.app.yaml
-    app2-submodule/
-        index.php
-```
+Note that disk paths in the `.platform.app.yaml` file are relative to the directory where that file lives by default.
 
-This puts your applications' files at a different path relative to your
-`.platform.app.yaml` files, so you'll also have to update your web
-configuration to match. For example, your first application's
-`.platform.app.yaml` file would include something like this:
-```
-web:
-    locations:
-        "/":
-            root: "app1-submodule"
-            # Or app1-submodule/path/to/webroot, if appropriate.
-            passthru: "/index.php"
-```
+This is the recommended approach for most configurations.
 
+### Explicit `source.root`
 
-### Multi-app Routes
+As an alternative, you may specify a `sources.root` key in a `.platform.app.yaml` file to override the "application root is where the file is" logic.  The `.platform.app.yaml` file may then live anywhere in the repository but use code from another directory.  Two separate `.platform.app.yaml` files may refer to the same directory if desired.
 
-If you setup the AngularJS `.platform.app.yaml` with `name: angular`, and the
-Drupal one `name: drupal`, you need to configure your `.platform/routes.yaml`
-like the following (names need to match):
+For example:
 
 ```yaml
-"https://backend.{default}/":
-    type: upstream
-    upstream: "drupal:http"
-"https://{default}/":
-    type: upstream
-    upstream: "angular:http"
-```
-
-This will result (if we consider we are on the `http://example.com` domain):
-* `http://backend.example.com` being served by the Drupal instance
-* `http://example.com/` and all the urls below it to be served by the AngularJS
-one.
+# .platform.app.yaml
 
-> **note**
-> Subdomain routes in development environments will be accessible using three dashes (---) instead of a dot (.), e.g:
-> http://backend---BRANCH-MACHINE_NAME-PROJECTID-.REGION.platformsh.site
-
-### SSH in each application
-
-You can SSH in any application that is deployed.
-
-If you are not sure how to construct the SSH URL, you can use the generic one
-provided by the Platform.sh management console:
+source:
+    root: restapp
+```
 
 ```bash
-ssh 3bdcrrivykjsm-master@git.eu-2.platform.sh
-
-The user name '3bdcrrivykjsm-master' resolves to multiple services, use one of
-those more specific names:
- - 3bdcrrivykjsm-master--angular
- - 3bdcrrivykjsm-master--drupal
-Received disconnect from 54.76.137.151: 14: No more auth methods available
-Disconnected from 54.76.137.151
+.platform/
+main/
+    .platform.app.yaml
+.platform.app.yaml
+restapp/
+    # Your code here
 ```
 
-## Example of a micro-service multi-app
+In this case, the `.platform.app.yaml` file in `main` does not specify a `sources.root`, and so will be built from the code in `main`.  The top-level `.platform.app.yaml` includes the YAML fragment above.  It will get built using the code in `restapp`, as if it were in that directory.
```

Comment:
```suggestion
In this case, the `.platform.app.yaml` file in `main` does not specify a `source.root`, and so will be built from the code in `main`.  The top-level `.platform.app.yaml` includes the YAML fragment above.  It will get built using the code in `restapp`, as if it were in that directory.
```